enclave_wrangler/new_enclave_api.py

```diff
@@ -556,15 +557,15 @@ def upload_concept_set_container(
     if assigned_informatician:
         d['parameters']['assigned_informatician'] = assigned_informatician
 
-    response: JSON_TYPE = post(api_name, d, validate_first)
+    response: Response = post(api_name, d, validate_first)
     if 'errorCode' in response:
         print(response, file=sys.stderr)
         print('If above error message does not say what is wrong, it is probably the case that the `concept_set_id` '
               f'already exists. You passed: {concept_set_id}')
     return response
 
 
-def make_request(api_name: str, data: Union[List, Dict] = None, validate=False, verbose=True) -> JSON_TYPE:
+def make_request(api_name: str, data: Union[List, Dict] = None, validate=False, verbose=True) -> Response:
```

Comment:
@Sigfried I had to change `make_request()` to return `requests.Response` instead of JSON. I need the full response for testing right now, as I'm checking the `status_code`. I should've just designed it that way in the beginning. Most of the changes in `new_enclave_api.py` are related to the the typing on this.